test/unit/helpers/xhr/FACTOR_CHALLENGE_email.js

```diff
@@ -2,42 +2,51 @@ define({
   "status": 200,
   "responseType": "json",
   "response": {
-    "stateToken": "01MMFqbYA4dwgGi0t5I78wIhcvHS1pOxKJE_B6jiUk",
-    "expiresAt": "2015-09-27T01:14:24.573Z",
+    "stateToken": "01bfpkAkRyqUZQAe3IzERUqZGOfvYhX83QYCQIDnKZ",
```

Comment:
nit: since you re changing for test data, you could also have this as `testToken`

Reply:
I want the ixd stateToken to have `01` in it, so that it evaluates the appState variable `isIdxStateToken` correctly.